lib/rex/post/meterpreter/extensions/powershell/powershell.rb

```diff
@@ -53,10 +53,10 @@ def import_file(opts={})
       request.add_tlv(TLV_TYPE_POWERSHELL_ASSEMBLY_SIZE, binary.length)
       request.add_tlv(TLV_TYPE_POWERSHELL_ASSEMBLY, binary)
       client.send_request(request)
-      return true
```

Comment:
As this is an API change, have all the callers of this API been updated to support this result type?

Reply:
Yes, as far as I could find, the only calls were:
- `lib/rex/post/meterpreter/ui/console/command_dispatcher/powershell.rb` (Updated)
- `modules/exploits/windows/local/tokenmagic.rb` (Didn't use the return value in the first place)
- `modules/exploits/windows/local/cve_2020_1337_printerdemon.rb` (Likewise, didn't use the return value in the first place)